src/components/MarketoForm/MarketoForm.js

```diff
@@ -4,8 +4,8 @@ import { css } from '@emotion/core';
 import './marketo.scss';
 import useMarketoForm from '../../hooks/useMarketoForm';
 
-const MarketoForm = ({ title, id, munchkinId, publishableKey }) => {
-  useMarketoForm(munchkinId, id, publishableKey);
+const MarketoForm = ({ title, id, munchkinId, publishableKey, thankYou }) => {
```

Comment:
```suggestion
const MarketoForm = ({ title, id, munchkinId, publishableKey, redirectLink }) => {
```

just wanted to rename this for reusability!